pkgs/by-name/rc/rcp/package.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv
+, buildPackages
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rcp";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "wykurz";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-5CqQwTJAQhO9mLfMan6JhNY3N2gfwR6wmGtVBYzVxuc=";
+  };
+
+  cargoHash = "sha256-sF7RjuVRNfJa3vw71S+BKIBLeWT6biekAE/56BsZYkw=";
+
+  checkFlags = [
+    # this test also sets setuid permissions on a test file (3oXXX) which doesn't work in a sandbox
+    "--skip=copy::copy_tests::check_default_mode"
+  ];
+
+  meta = with lib; {
+    description = "Tools to efficiently copy, remove and link large filesets.";
```

Descriptions should not end with a period[^1].
```suggestion
    description = "Tools to efficiently copy, remove and link large filesets";
```

[^1]: https://github.com/NixOS/nixpkgs/blob/f863c227fe1d633a615af33861c166d9dca8ad0b/pkgs/README.md#meta-attributes